docs/translations.md

```diff
@@ -1,43 +1,777 @@
+  also avoids the spelling differences between American and British English.
+- **[Localization][]**: Adapting software to a specific locale. This includes translating
+  text and using locale-specific formats. This is often abbreviated **L10n**, in a
+  similar way to i18n. A capital "L" avoids confusing a lowercase "l" with an
+  uppercase "I".
+- **[Locale][]**: Shared cultural conventions that appear in a user interface. This
+  can include language, letters, and currencies. It can also include formats for prices,
+  numbers, dates, and times. For example, American English can be a locale. This locale
+  uses the English alphabet in left-to-right text. Prices are in US Dollars (USD, "$5.99").
+  Numbers use commas as thousands separators (525,960 minutes in a year). The first
+  number in a short date is the month ("9/12/2023" for September 12, 2023).
+  English-speaking users in other regions share some of these conventions. Other
+  conventions, like currency and date formats, will vary.
+- **[Region][]**: An [administrative division][] that can be the basis for a
+  locale. Mozilla prefers to use "Region" instead of "**Country**" when talking about
+  localization. A region can include areas within a country, as well as areas that
+  span countries.
+- **[Language][]**: A spoken or written system of communication. English is a broad
+  language category. A language can be specific to a region, such as German spoken in
+  Switzerland. In localization, the written system can be important as well. For
+  example, Chinese can vary between mainland China and other regions. It can also
+  use Simplified or Traditional Chinese script when written.
+- **[Translation][]**: The process of adapting language strings or patterns to a
+  second language. This is one aspect of Localization. At Mozilla, American English is
+  the source language for interface text.
+
+[Internationalization]: https://en.wikipedia.org/wiki/Internationalization_and_localization
+[Language]: https://en.wikipedia.org/wiki/Language
+[Locale]: https://en.wikipedia.org/wiki/Locale_(computer_software)
+[Localization]: https://en.wikipedia.org/wiki/Internationalization_and_localization
+[Region]: https://en.wikipedia.org/wiki/Region
+[Translation]: https://en.wikipedia.org/wiki/Translation
+[administrative division]: https://en.wikipedia.org/wiki/Administrative_division
+
+## Standards for Identifiers
+
+There are internet standard identifiers for regions, languages, locales, and currency.
+In most cases, Mozilla uses the standard identifiers. There are exceptions for common
+usage and legacy cases. Relay users should follow the standards, unless Mozilla has an
+established exception.
+
+- **Region**: [ISO 3166][] is a multi-part standard for identifying countries,
+  territories, and subdivisions. The [ISO 3166-1 alpha-2][] code (two uppercase
+  letters) is common as a region identifier.
+  - Mozilla uses the ISO 3166-1 alpha-2 code for a region when available. For example,
```

Comment:
(question, blocking) I want to clarify if "Mozilla" referenced in this section means Pontoon or Firefox?